src/Oscoin/API/Client.hs

```diff
@@ -6,15 +6,15 @@ import           Oscoin.Prelude
 
 import           Oscoin.API.Types
 import           Oscoin.Crypto.Hash (Hashed)
-import qualified Oscoin.Data.RadicleTx as RadicleTx
+import           Oscoin.Data.Tx (Tx, TxPayload)
 
 class Monad m => MonadClient c m where
-    submitTransaction :: RadicleTx.RadTx c -> m (Result (TxSubmitResponse c (RadicleTx.RadTx c)))
+    submitTransaction :: Tx c -> m (Result (TxSubmitResponse c (Tx c)))
 
     -- | Returns an error result if a transaction with the given hash
     -- was not found.
-    getTransaction :: Hashed c (RadicleTx.RadTx c) -> m (Result (TxLookupResponse c))
+    getTransaction :: Hashed c (Tx c) -> m (Result (TxLookupResponse c))
 
     -- | Returns an error result if a value with the given key was not
     -- found.
-    getState :: Proxy c -> [Text] -> m (Result RadicleTx.Value)
+    getState :: Proxy c -> [Text] -> m (Result (TxPayload c (Tx c)))
```

Comment:
This will be `StateVal` eventually, but for now should probably just be `ByteString`

Reply:
Agreed, the spirit was to change as little things as possible, as we discussed 😁